diff --git a/hw/kdrive/fbdev/meson.build b/hw/kdrive/fbdev/meson.build
--- a/hw/kdrive/fbdev/meson.build
+++ b/hw/kdrive/fbdev/meson.build
@@ -1,4 +1,5 @@
 srcs = [
+    'vibeos-popcount.c',
     'fbdev.c',
     'fbinit.c',
     '../../stubs/ddxBeforeReset.c',
diff --git a/hw/vfb/meson.build b/hw/vfb/meson.build
--- a/hw/vfb/meson.build
+++ b/hw/vfb/meson.build
@@ -1,4 +1,5 @@
 srcs = [
+    'vibeos-popcount.c',
     'InitInput.c',
     'InitOutput.c',
     '../../mi/miinitext.c',
